components/more_direct_channels/last_post_at.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {FunctionComponent} from 'react';
+import moment from 'moment-timezone';
+import {FormattedMessage} from 'react-intl';
+
+type LastPostAtProps = {
```

Comment:
[minor] Since this is a local type, maybe just `Props` would be a bit more concise?